src/Simulation/Algorithms/COE/RV2COE.hpp

```
@@ -0,0 +1,189 @@
+        double sme = (v_mag * v_mag) / 2 - mu / r_mag;
+    
+
+        // Semi-major axis & Semi-latus rectum
+        double a, p;
+        if (glm::abs(sme) > EPSILON)    a = -mu / (2.0 * sme);
+        else                            a = std::numeric_limits<double>::infinity();
+        p = (h_mag * h_mag) / mu;
+
+
+        // Inclination
+        double cos_incl = h.z / h_mag;
```

Clang (14.0.6)

[warning] variable 'cos_incl' is not initialized

(cppcoreguidelines-init-variables)